c_tests/tests/blockmap.c

```diff
@@ -1,3 +1,6 @@
+// Compiler:
```

Comment:
Shouldn't we at least check for `status: success`? At the moment these tests don't test anything at all.

Reply:
Oops. Yes, that was my intention!

Reply:
Docs say:

> If not specified, defaults to success.

Assuming that's correct?

Reply:
I put an `assert(0)` in the test and it flagged as a failure, so I think the docs are correct.

Reply:
I stand corrected!